### PR TITLE
Prebid Core: add 'targetingControls.addTargetingKeys' configuration option

### PR DESCRIPTION
**Pull request by dgirardi**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

﻿This adds 'addTargetingKeys' as a config option that means 'allow all default targeting keys, plus these'.

Addresses issue: https://github.com/prebid/Prebid.js/issues/7249

Documentation changes: https://github.com/prebid/prebid.github.io/pull/3401

### CODE REVIEW

**Comment by Rothalack** [comment]
This is working in my testing. Also tested the error if both are set and that's working. I think this is good to go.

Will we want a second review on this? @ChrisHuie @robertrmartinez

**Review by Rothalack — APPROVED** [review]

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**
